File: Modelo/Movimiento.cs
using System;

namespace punto_de_venta_C_.Modelo
{
    public class Movimiento
    {
        public enum Tipo
        {
            VENTA, RETIRO, FONDO_INICIAL
        }

        private Tipo tipo;
        private string descripcion;
        private double monto;
        private DateTime fecha;
        private string usuario;

        public Movimiento(Tipo tipo, string descripcion, double monto, DateTime fecha, string usuario)
        {
            this.tipo = tipo;
            this.descripcion = descripcion;
            this.monto = monto;
            this.fecha = fecha;
            this.usuario = usuario;
        }

        public Tipo getTipo() { return tipo; }
        public void setTipo(Tipo tipo) { this.tipo = tipo; }
        public string getDescripcion() { return descripcion; }
        public void setDescripcion(string descripcion) { this.descripcion = descripcion; }
        public double getMonto() { return monto; }
        public void setMonto(double monto) { this.monto = monto; }
        public DateTime getFecha() { return fecha; }
        public void setFecha(DateTime fecha) { this.fecha = fecha; }
        public string getUsuario() { return usuario; }
        public void setUsuario(string usuario) { this.usuario = usuario; }

        public bool esIngreso()
        {
            return tipo == Tipo.VENTA || tipo == Tipo.FONDO_INICIAL;
        }
    }
}

File: Modelo/Movimiento.py
from enum import Enum

class Tipo(Enum):
    VENTA = 'VENTA'
    RETIRO = 'RETIRO'
    FONDO_INICIAL = 'FONDO_INICIAL'

class Movimiento:
    def __init__(self, tipo=None, descripcion='', monto=0.0, fecha=None, usuario=''):
        self._tipo = tipo
        self._descripcion = descripcion
        self._monto = monto
        self._fecha = fecha
        self._usuario = usuario

    def getTipo(self):
        return self._tipo

    def setTipo(self, tipo):
        self._tipo = tipo

    def getDescripcion(self):
        return self._descripcion

    def setDescripcion(self, descripcion):
        self._descripcion = descripcion

    def getMonto(self):
        return self._monto

    def setMonto(self, monto):
        self._monto = monto

    def getFecha(self):
        return self._fecha

    def setFecha(self, fecha):
        self._fecha = fecha

    def getUsuario(self):
        return self._usuario

    def setUsuario(self, usuario):
        self._usuario = usuario

    def esIngreso(self):
        # We need to support enum checks. Sometimes in Python, self._tipo is the enum member or its value.
        # It's safest to check both.
        tipo_val = self._tipo
        if hasattr(tipo_val, 'name'):
            tipo_val = tipo_val.name
        
        return tipo_val == 'VENTA' or tipo_val == 'FONDO_INICIAL' or tipo_val == Tipo.VENTA or tipo_val == Tipo.FONDO_INICIAL
